src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -6,13 +6,13 @@
 import seedu.address.model.Model;
 
 /**
- * Lists all persons in the address book to the user.
+ * Lists all pets in the address book to the user.
  */
 public class ListCommand extends Command {
 
     public static final String COMMAND_WORD = "list";
 
-    public static final String MESSAGE_SUCCESS = "Listed all persons";
+    public static final String MESSAGE_SUCCESS = "Listed all pet";
```

Comment:
Should it be "Listed all _pets_" instead?

Reply:
Yup agreed. Thanks for catching this.